packages/http/src/index.ts

```diff
@@ -25,7 +25,7 @@ const createInstance = <LoaderInput>(
   return factory<IHttpOperation, IHttpRequest, IHttpResponse, IHttpConfig, LoaderInput>(
     { mock: true },
     {
-      loader: filesystemLoaderInstance,
+      loader: new FilesystemLoader(),
```

Comment:
Does that fix this 'singleton' bug? Would be nice to write a unit test to cover this fix (test fails w/o this fix and passes with the fix - standard TDD).

Reply:
Added in 6fd666b